tests/e2e-demos/e2e-demo.go

```diff
@@ -203,14 +203,14 @@ var _ = framework.E2ESuiteDescribe(Label("e2e-demo"), func() {
 							envbinding, err := fw.IntegrationController.GetSnapshotEnvironmentBinding(application.Name, namespace, env)
 							Expect(err).ShouldNot(HaveOccurred())
 							Expect(envbinding != nil).To(BeTrue())
-							GinkgoWriter.Printf("The EnvironmentBinding %s is created\n", envbinding.Name)
+							GinkgoWriter.Printf("The SnapshotEnvironmentBinding %s is created\n", envbinding.Name)
 							return true
 						}
 
 						snapshot, err = fw.IntegrationController.GetApplicationSnapshot("", application.Name, namespace, component.Name)
 						Expect(err).ShouldNot(HaveOccurred())
 						return false
-					}, timeout, interval).Should(BeTrue(), "time out when waiting for release created")
+					}, timeout, interval).Should(BeTrue(), "time out when trying to check if SnapshotEnvironmentBinding is created")
```

Comment:
include snapshot name and env, so that it is easier to find the issue.

Reply:
Sure! Thank you!